components/MessageOverlay.tsx

```diff
@@ -60,6 +65,14 @@ export const MessageOverlay: React.FC<MessageOverlayProps> = props => {
             styles={Theme.MessageOverlayStyles.button}
           />
         ) : null}
+        {!props.children && props.onCustomBtnTxt ? (
```

Comment:
If we pass onCancel and onCustomBtnTxt both, are we expecting to render both the buttons?
If yes, can you please attach screenshot how does it look?
if no, then we have to make change that it MessageOverlay renders only one button, the text and action of the button is received as props